### ISSUE TITLE
Add a project for testing the UI.Xaml.Build

### ISSUE THREAD

**Issue by circler3** [aloisdeniel/Mwm#2]
I found I cannot load any symbols for UI.Xaml.Build project since the project reference is *target* file. It is not easy to debug the project. 
Maybe a testing project could be added to current solution. Or is it possible that UI.Xaml.Build can be referenced by the *sample* project as the same as *Mvm.UI* project.

**Comment by aloisdeniel** [comment]
Yes custom build task is a pain to debug (I have to reload the entire solution at each modification). But yes, a test project is definitively a must have for testing the XamlGenerator.
